src/plugin.ts

```diff
@@ -12,7 +13,14 @@ const plugin = (): Array<Plugin> => {
   let resolvedConfig: ResolvedConfig
   let platformEntryPath: string
   return [
-    netlifyPlugin(),
+    netlifyPlugin({
+      additionalStaticPaths: (config) =>
+        glob
+          .sync('*.js', {
+            cwd: path.resolve(config.root, 'dist/client'),
+          })
+          .map((file) => `${config.base}${encodeURIComponent(file)}`),
+    }),
```

Comment:
Thanks @nickytonline, that's awesome. I think I'm going to call the env var CLIENT_DIR so that it doesn't get confused with the `--outDir` Vite param, which might be set in the SSR build step. Cheers.